app/assets/sass/modules/_buttons.scss

```diff
@@ -6,11 +6,17 @@
   cursor: pointer;
   text-decoration: underline;
   color: $link-colour;
+  box-shadow: none;
```

Comment:
why are we adding more custom css.

Reply:
We need UpScan File upload render correctly and we have added same for Challenge